kotlin/android/build.gradle.kts

```diff
@@ -0,0 +1,104 @@
+plugins {
+    kotlin("jvm")
+    id("com.google.protobuf")
+    id("maven-publish")
+}
+
+dependencies {
+    // Descriptor.proto is included because there is no lite version of common protos:
+    // https://github.com/protocolbuffers/protobuf/issues/1889
+    protobuf(files("./google-protos"))
```

Comment:
Is this the one that includes `descriptor.proto`? Not sure if the google-protos lib was auto generated due to this.. am thinking if we should only pull `descriptor.proto` if at all possible?

Reply:
We need all of the protos in there. The three in `api` are redundant, since they are in the main protos directory, but I couldn't get multiple file includes to work.

Reply:
I updated this directory to only contain `descriptor.proto`. We don't need to include the others now that we don't need to exclude webhook.